packages/workato-adapter/src/adapter.ts

```diff
@@ -140,14 +144,54 @@ export default class WorkatoAdapter implements AdapterOperations {
    */
   @logDuration('deploying account configuration')
   // eslint-disable-next-line class-methods-use-this
-  async deploy(): Promise<DeployResult> {
-    throw new Error('Not implemented.')
+  async deploy({ changeGroup }: DeployOptions): Promise<DeployResult> {
+    // resolving workato and cross-service
+    const resolvedChanges = await Promise.all(
+      changeGroup.changes.map(change => resolveChangeElement(change, getLookUpName, resolveValuesRecipeCodeWrapper))
+    )
+
+    log.debug('GroupID - ', changeGroup.groupID)
+    const runner = this.createFiltersRunner()
+    await runner.preDeploy(resolvedChanges)
+
+    let deployResult = { appliedChanges: [], errors: [] } as DeployResult
+    if (changeGroup.groupID === DEPLOY_USING_RLM_TYPE) {
+      const [instanceChanges, nonInstanceChanges] = _.partition(resolvedChanges, isInstanceChange)
+      if (nonInstanceChanges.length > 0) log.error(`Get non-InstanceChanges in ${DEPLOY_USING_RLM_TYPE} group`)
```

Comment:
1. please always use opening and closing `{}`, it's bad practice to use these as they can very easily turn into incorrect code on edits
```suggestion
      if (nonInstanceChanges.length > 0) {
        log.error(`Get non-InstanceChanges in ${DEPLOY_USING_RLM_TYPE} group`)
      }
```
2. if we're logging let's make the log more informative and include the elem ids that we're not going to deploy